### PR TITLE
revert 1.10 verify image bump

### PR DESCRIPTION
**Pull request by BenTheElder**
it looks like the golang version changed in the last bump @ixdy 

since 1.10 is out of support (but we are probably picking in one last fix for a big we picked into the last patch), we can just revert the image change and later drop 1.10 after that fix is in. xref #10742

### CODE REVIEW

**Comment by BenTheElder** [comment]
xref: https://github.com/kubernetes/kubernetes/pull/72860

File: scenarios/kubernetes_verify.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 
 VERSION_TAG = {
-    '1.10': '1.10-v20181218-db74ab3f4',
+    '1.10': '1.10-v20180221-3655e24ae',
```
Comment (ixdy) [inline scenarios/kubernetes_verify.py]: Both versions have go1.9.3 installed...

Comment (BenTheElder) [reply scenarios/kubernetes_verify.py]: whoooo knows. this seems to fix something though /shrug

Comment (BenTheElder) [reply scenarios/kubernetes_verify.py]: either way we should be deleting this now-ish, just holding out for a possible last patch

**Pull request merged**